notebooks/DemoNotebook_statistics.ipynb

```diff
@@ -0,0 +1,205 @@
+    "import seaborn as sns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Read in the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_dict = mn.InputOutput.get_input_dir(\n",
+    "    \"/home/inga/projects/moralization-project/moralization/data/\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Report instances of occurence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# i would guess from here one can get the correlation table somehow.\n",
+    "# just df.corr() yields unsatisfactory results..\n",
+    "\n",
+    "# these DataFrames can now easily be saved as a csv file.\n",
+    "df_instances = mn.analyse.AnalyseOccurence(data_dict, mode=\"instances\").df\n",
+    "df_instances.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this df can now easily be filtered.\n",
+    "df_instances.loc[\"KAT2Subjektive_Ausdrcke\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Report the spans of the instances."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_spans = mn.analyse.AnalyseOccurence(data_dict, mode=\"spans\").df\n",
+    "df_spans.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Further exploratory analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mn.analyse.get_overlap_percent(\n",
+    "    \"Forderer:in\",\n",
+    "    \"Neutral\",\n",
+    "    data_dict,\n",
+    "    \"test_data-trimmed_version_of-Gerichtsurteile-neg-AW-neu-optimiert-BB\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = mn.analyse.get_percent_matrix(\n",
+    "    data_dict, \"test_data-trimmed_version_of-Gerichtsurteile-neg-AW-neu-optimiert-BB\"\n",
+    ")\n",
+    "plt.figure(figsize=(16, 16))\n",
+    "ax = sns.heatmap(df, cmap=\"cividis\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_small = mn.analyse.get_percent_matrix(\n",
+    "    data_dict,\n",
+    "    \"test_data-trimmed_version_of-Gerichtsurteile-neg-AW-neu-optimiert-BB\",\n",
+    "    [\n",
+    "        \"Appell\",\n",
+    "        \"Adresassat:in\",\n",
+    "        \"Forderer:in\",\n",
+    "        \"soziale Gruppe\",\n",
+    "        \"Benefizient:in\",\n",
+    "        \"Neutral\",\n",
+    "        \"Institution\",\n",
+    "        \"Expression\",\n",
```

Comment:
This gives an error for me using the test dataset, but if I remove `"Adresassat:in"` and  `"Expression"` (I also don't see these values in the previous plot) then it works - maybe the test dataset changed?